server/ServerFSM.cpp

```diff
@@ -622,7 +623,63 @@ sc::result Idle::react(const Error& msg) {
 }
 
 sc::result Idle::react(const Hostless&) {
-    return transit<MPLobby>();
+    TraceLogger(FSM) << "(ServerFSM) Idle.Hostless";
+    std::string autostart_load_filename = GetOptionsDB().Get<std::string>("load");
+    if (autostart_load_filename.empty())
+        return transit<MPLobby>();
+
+    if (GetOptionsDB().Get<int>("network.server.conn-human-empire-players.min") > 0) {
+        throw std::invalid_argument("Autostart load file was choosed but the server wasn't allowed to play game without connected players");
```

Comment:
A save file to load and autostart in hostless mode was specified, but the server has a non-zero minimum number of connected players, so cannot be started without a connected player.

Reply:
might be better to make a customized exception for this, and catch it in dmain and output something translated from the stringtable.

Reply:
Fixed.